jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -55,49 +75,156 @@ public class PolicyExecutor {
 
     /** The Constant logger. */
     private static final Logger logger = LoggerFactory.getLogger(PolicyExecutor.class);
-    
+
     /** The is resource filter exists. */
-    private Boolean isResourceFilterExists = Boolean.FALSE;
+    //  private Boolean isResourceFilterExists = Boolean.FALSE;
+
+    private List<Map<String, String>> resources = new ArrayList<>();
+    private String targetType;
+    private String source;
+    private List<Map<String, String>> policyWiseParamsList = new ArrayList<>();
+    private static final int POLICY_THREAD_POOL_SIZE = 50;
+    private static final String POLICY_DONE_SNS_TOPIC_ARN = "POLICY_DONE_SNS_TOPIC_ARN";
+
+    public PolicyExecutor (String jsonString) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();

```

Comment:
I see you create another instance of mapper below. Why don't you just create private object variable and reuse it everywhere?